File: src/main/app/com/leetcode/BasicDataTypes/1523_CountOddNumbersInAnIntervalRange.java
package com.leetcode.BasicDataTypes;

class Solution174 {
    public int countOdds(int low, int high) {
        int result = 0;
        if (low % 2 == 0) low += 1;
        for(var i = low; i<=high; i+=2) {
            result++;
        }
        return result;
    }
}

File: solutins-using-java-python/src/main/app/com/leetcode/BasicDataTypes/1523_CountOddNumbersInAnIntervalRange.kt
package com.leetcode.BasicDataTypes

class `1523_CountOddNumbersInAnIntervalRange` {
    fun countOdds(low: Int, high: Int): Int {
        val fromVal: Int = if (low % 2 == 0) low + 1 else low
        var count = 0
        for (i in fromVal..high step 2) {
            count++
        }

        return count
    }
}
